pipelines/base/peribolos-dump-config.yaml

```diff
@@ -14,7 +14,29 @@ spec:
   - name: GIT_NAME
     type: string
   steps:
-    - name: dump-and-push-config
+    - name: dump-config
+      image: quay.io/skopecky/peribolos-fix
+      volumeMounts:
+        - mountPath: /var/lib/shared
+          name: shared-data
+      env:
+        - name: GITHUB_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: peribolos-$(params.INSTALLATION_ID)
+              key: token
+        - name: ORG_NAME
+          valueFrom:
+            secretKeyRef:
+              name: peribolos-$(params.INSTALLATION_ID)
+              key: orgName
+      script: |
+        #!/bin/bash
+
+        echo -n $GITHUB_TOKEN > token
+        echo "Dumping org config ..."
+        peribolos --dump $ORG_NAME --github-token-path token > /var/lib/shared/peribolos.yaml
```

Comment:
Do we need a script here? I mean, we're building a custom image anyways.. so why don't we just make the `peribolos` binary image's entrypoint? and then in the step spec you just pass it the arguments via `args`.

Reply:
I agree it is a better solution to use `args` , I tried to to this earlier but since the peribolos `--dump` outputs the config to `stdout` I couldn't find a way to redirect `stdout` from the step to a file for peribolos binary as the entry point. Do you know of a way how to do this?

Reply:
I didn't find a good solution either. Let's stick to a tiny script for now.